Launcher/Classes/ApplicationStateHandler.swift

```diff
@@ -15,11 +15,13 @@ class ApplicationStateHandler {
         case debugState = "debugCheckboxState"
         case cucumberProfile = "cucumberProfile"
         case additionalRunParameters = "additionalRunParameters"
+        case deviceIP = "deviceIP"
+        case bundleID = "bundleID"
     }
     
-    var simulatorRadioButtonState: Int {
+    var simulatorRadioButtonState: Bool? {
```

Comment:
Why is this optional? `bool(forKey:)` returns a `Bool`: https://developer.apple.com/documentation/foundation/userdefaults/1416388-bool

Reply:
Thanks for pointing this out. It was actually causing a bug for a new user.